### ISSUE TITLE
Support Kuberenetes PV/PVC's that don't have file ownership

### ISSUE THREAD

**Issue by docktermj** [senzing-garage/docker-init-container#140]
On Kubernetes systems (i.e. AWS AKS) with certain PV types, the following is seen when running `init-container.py`

```console
cp: failed to preserve ownership for '/opt/senzing/./g2/sdk': Operation not permitted
cp: failed to preserve ownership for '/opt/senzing/./g2': Operation not permitted
```

The first ramification of this is the use of the `--archive` attribute in a `cp` command.

https://github.com/Senzing/docker-installer/blob/603cbaad51bb79304f05bf90b674e32810db8c7c/Dockerfile#L69

See: https://github.com/Senzing/docker-installer/issues/12

This also manifests itself in errors seen in 

https://github.com/Senzing/docker-init-container/blob/2f2b3086341cd0cda9e0e4bcc19855cc026ecc79/init-container.py#L911-L949

`init-container.py` has log errors in file attribute changes and complete processing.

In other words, use `try:  exception:` around all `os.chown()` and `os.chmod()`

**Comment by docktermj** [comment]
Deprecated
